bindings/cs/CMakeLists.txt

```diff
@@ -1 +1,29 @@
-add_subdirectory(rl.net.native)
+if (rlclientlib_BUILD_DOTNET)
+  set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+  include(FindDotnet)
+
+  add_subdirectory(rl.net.native)
+
+  add_custom_target(rl.net

```

Comment:
Can you add the sources to these targets so when a VS solution is generated they show up?

Reply:
Done. At that point it makes sense to break them out into separate CMakeLists.txt files. 

It is somewhat of an impedance mismatch between CMake and the dotnet toolchain, as .Net developers will not naturally think to add new sources into the CMakeLists.txt file. That was one of the big gains in Sdk-style projects: including all source in the tree by default.